src/intervals/arithmetic.jl

```diff
@@ -155,6 +155,25 @@ end
 
 //(a::Interval, b::Interval) = a / b    # to deal with rationals
 
+if VERSION >= v"0.6.0-dev"
```

Comment:
There should be a PR linked to in NEWS or otherwise not too hard to find, and there's the `contrib/commit-name.sh` script you can use